utils_cv/detection/dataset.py

```diff
@@ -20,12 +20,48 @@
 from .bbox import AnnotationBbox
 from .mask import binarise_mask
 from .references.utils import collate_fn
-from .references.transforms import RandomHorizontalFlip, Compose, ToTensor
+from .references.transforms import Compose, ToTensor
 from ..common.gpu import db_num_workers
 
 Trans = Callable[[object, dict], Tuple[object, dict]]
 
 
+def _flip_keypoints(kps, width, hflip_inds):
```

Comment:
[minor] kps should probably be called keypoints for consistency

Reply:
Renamed